Add `osu! Philippines Fanart Contest 2024` article (#12304)
ppy/osu-wiki: wiki/Contests/PHFC/2024/en.md

---
tags:
  - phfc
  - phfc 2024
---

> # osu! Philippines Fanart Contest 2024 — "Kasiyahan sa Kapaskohan"

Reviewer: em dash isn't used when separating subtitle from title

```suggestion
# osu! Philippines Fanart Contest 2024 - "Kasiyahan sa Kapaskohan"
```

![](img/logo.png)

> The **osu! Philippines Fanart Contest 2024 — "Kasiyahan sa Kapaskohan"** (***PHFC2024***) is a fanart contest made for the Philippine osu! community featuring its four mascots. It is themed after the Christmas season.

Reviewer 2: ```suggestion
The **osu! Philippines Fanart Contest 2024 — "Kasiyahan sa Kapaskohan"** (***PHFC2024***) is a fanart contest run by the Philippine osu! community featuring its four mascots. It is themed after the Christmas season.
```

Reviewer 3: I believe the issue here was the use of "made". Removed that word and shuffled the sentencse a little bit. I believe the current wording reads nicer as well.

> The **osu! Philippines Fanart Contest 2024 — "Kasiyahan sa Kapaskohan"** (***PHFC2024***) is a fanart contest for the Philippine osu! community. The contest features the four osu! Philippines mascots and is themed after the Christmas season.

Reviewer: ```suggestion
The **osu! Philippines Fanart Contest 2024 - "Kasiyahan sa Kapaskohan"** (***PHFC2024***) is a fanart contest for the Philippine osu! community. The contest features the four osu! Philippines mascots and is themed after the Christmas season.
```

## Schedule

| Event | Timestamp |
| --: | :-- |

> | Submission Period | 2024-10-11/2024-11-08 (15:59 UTC) |
> | Entry Screening | 2024-11-08/2025-11-15 |
> | Public Voting | 2024-11-15/2024-11-29 |

Reviewer: ```suggestion
| Submission period | 2024-10-11/2024-11-08 (15:59 UTC) |
| Entry screening | 2024-11-08/2025-11-15 |
| Public voting | 2024-11-15/2024-11-29 |
```

## Prizes

> The contest offers a minimum prize pool worth PHP 2,500.00. (*Amount is currently tentative.*)

Reviewer 2: ```suggestion
The contest offers an initial prize pool of 2,500.00 PHP that may subject to increase on incoming sponsors.
```

Reviewer: ```suggestion
The contest offers an initial prize pool of 2,500.00 PHP that is subject to increase via sponsors.
```

Reviewer 3: ```suggestion
The contest offers an initial prize pool of 2,500.00 PHP that is subject to increase via donations.
```

| Placing | Prize(s) |
| :-: | :-- |
| ![Gold crown](crown-gold.png "1st place") | 50% of the prize pool, unique profile badge (*proposed*) |
| ![Silver crown](crown-silver.png "2nd place") | 35% of the prize pool |
| ![Bronze crown](crown-bronze.png "3rd place") | 15% of the prize pool |

## Organisation

The osu! Philippines Mascots Fanart Contest 2024 is run by the following staff:

| Position | Member(s) |
| :-- | :-- |
| Host | ::{ flag=NL }:: [Albionthegreat](https://osu.ppy.sh/users/9853595), ::{ flag=PH }:: [SurfChu85](https://osu.ppy.sh/users/4469895), ::{ flag=PH }:: [topecnz](https://osu.ppy.sh/users/2103927) |

> | Contest Moderator | ::{ flag=NL }:: [Albionthegreat](https://osu.ppy.sh/users/9853595), ::{ flag=CN }:: [Sakura006](https://osu.ppy.sh/users/10365024), ::{ flag=PH }::  [topecnz](https://osu.ppy.sh/users/2103927) |

Reviewer: ```suggestion
| Contest moderator | ::{ flag=NL }:: [Albionthegreat](https://osu.ppy.sh/users/9853595), ::{ flag=CN }:: [Sakura006](https://osu.ppy.sh/users/10365024), ::{ flag=PH }::  [topecnz](https://osu.ppy.sh/users/2103927) |
```

| Developer | ::{ flag=PH }:: [-Ichinose](https://osu.ppy.sh/users/12635292), ::{ flag=PH }:: [topecnz](https://osu.ppy.sh/users/2103927) |
| Statistician | ::{ flag=NL }:: [Albionthegreat](https://osu.ppy.sh/users/9853595), ::{ flag=PH }:: [SurfChu85](https://osu.ppy.sh/users/4469895) |

> | Wiki Editor | ::{ flag=PH }:: [SurfChu85](https://osu.ppy.sh/users/4469895) |

Reviewer: ```suggestion
| Wiki editor | ::{ flag=PH }:: [SurfChu85](https://osu.ppy.sh/users/4469895) |
```

## Links

> - Contest forum post
> - Contest entry submission

Reviewer: ```suggestion
- Submission form
- Discussion thread
```

- [osu! Philippines Discord server](https://discord.gg/officialosuph)

## Rulesets

> - By submitting your artwork to this contest, you give osu! Philppines the permission to use it for any promotional material for its activities, not limited to events, promotions, and the like. You will retain all of the rights to your work and you are permitted to sell or distribute it.

Reviewer 2: ```suggestion
- By submitting your artwork, you give osu! Philppines the permission to use it for any promotional material for its activities including (but not limited to) events, promotions, and the likes. You will retain all of the rights to your work, and you will be permitted to sell or distribute it.
```

Reviewer: "like" in "and the like" doesn't take a plural form

> - Participants must have the Philippine flag on their profiles.

Reviewer: ```suggestion
- Participants must have the Philippines flag on their profiles.
```

- All entries MUST follow the contest theme and contain AT LEAST two of osu! Philippines' community mascots, one of which must be Primrea. You may look at these links for your reference:

>   - Primrea, the osu! Philippines/#filipino mascot

Reviewer: assuming this is referring to a chat channel

```suggestion
  - Primrea, the osu! Philippines/`#filipino` mascot
```

  - Primreadon, the osu!taiko PH mascot
  - Punica, the osu!catch PH mascot
  - [Krysta, the osu!mania PH mascot](https://up.topecnz.net/maniaph_krysta_presskitv5.pdf)
- All entries MUST be suitable for viewing by all ages.
- Your submission must be your own original work. Redrawing and composite works are not allowed.
- You can only submit ONE entry. You may reupload your entry for adjustments as many times as you want, but once the submission deadline has passed, only your latest submission will be considered.
- Generative art programs or methods (like AI) are not permitted.

> - Participants must keep a full file containing all of their entry's layers available. The staff will only ask you for this if they think it's needed to ascertain any claims regarding plagiarism or the use of generative art tools. If you do not/can not provide this within a timely manner when asked, your entry will be disqualified.

Reviewer: ```suggestion
- Participants must keep a full file containing all of their entry's layers available. The staff will only ask you for this if they think it is needed to ascertain any claims regarding plagiarism or the use of generative art tools. If you do not/cannot provide this within a timely manner when asked, your entry will be disqualified.
```

> - Participants must keep a full file containing all of their entry's layers available. The staff will only ask you for this if they think it is needed to ascertain any claims regarding plagiarism or the use of generative art tools. If you do not/cannot provide this within a timely manner when asked, your entry will be disqualified.

Reviewer 2: ```suggestion
- Participants must keep a full file containing all of their entry's layers available. The staff will only ask you for this if they think it is needed to ascertain any claims regarding plagiarism or the use of generative art tools. If you do not/can not provide this within a timely manner when asked, your entry will be disqualified.
```

Reviewer: wiki uses "cannot"

> - Artist attribution (names, social media handles, etc.) must be limited to no more than three lines total and confined to any corner of your entry.

Reviewer 2: ```suggestion
- Artist attribution (names, social media handles, etc.) must be limited to no more than three lines total and be confined in a corner.
```

Reviewer: this is likely copied from the wording in official art contests, but if removing "you" is the goal here then this would sound ok enough

```suggestion
- Artist attribution (names, social media handles, etc.) must be limited to no more than three lines total and be confined to any corner of the entry.
```

> - Entries MUST meet the resolution requirements for social media banners, specifically Facebook's and Discord's. You may check [this template](https://up.topecnz.net/fanart-sample-template.png) as reference.

Reviewer: ```suggestion
- Entries MUST meet the resolution requirements for social media banners, specifically Facebook and Discord. You may check [this template](https://up.topecnz.net/fanart-sample-template.png) as reference.
```

> - Entries MUST meet the resolution requirements for social media banners, specifically Facebook's and Discord's. You may check [this template](https://up.topecnz.net/fanart-sample-template.png) as reference.

Reviewer: please put the image into `wiki/Contests/PHFC/2024/img/` and link it as `img/image-name.png`

Reviewer 3: Done 👍🏻

> - Entries MUST meet the resolution requirements for social media banners, specifically Facebook and Discord's. You may check [this template](https://up.topecnz.net/fanart-sample-template.png) as reference.

Reviewer 2: ```suggestion
- Entries MUST meet the resolution requirements for social media banners, specifically that of Facebook and Discord. You may check [this template](https://up.topecnz.net/fanart-sample-template.png) as a reference.
```

  - The main subjects of your artwork must appear within the Facebook group page banner's dimensions.

>   - *It is recommended that you submit your entries with as high of a resolution that you can provide. This is limited this to a 4K resolution (3840 x 2160) size.*

Reviewer: ```suggestion
  - *It is recommended that you submit your entries with as high of a resolution that you can provide. This is limited to a 4K resolution (3840x2160).*
```
